config/migrations/20232112140001-remove-ontwerpdecreet-vlaamse-regering.sparql

```diff
@@ -0,0 +1,6 @@
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+
+DELETE WHERE {
+   GRAPH ?g { ?s mu:uuid "8a1e048a-4b55-4a19-b1c0-c85dba09a15c";
```

Comment:
Since these concepts are only in the public graph, I'd prefer `<http://mu.semte.ch/graphs/public>` to be used directly. Updated this during review